markdowns/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.md

```diff
@@ -0,0 +1,190 @@
+---
+layout: tutorial
+colab: https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/19_Text_to_Image_search_pipeline_with_MultiModal_Retriever.ipynb
+toc: True
+title: "Text-to-Image search pipeline with MultiModal Retriever"
+last_updated: 2022-11-03
+level: "intermediate"
+weight: 95
+description: Use a MultiModalRetriever to build a cross modal search pipeline.
+category: "QA"
+aliases: ['/tutorials/multimodal']
+---
+    
+
+**Level**: Intermediate
+
+**Time to complete**: 20 minutes
+
+**Prerequisites**: Prepare the Colab environment (see links below).
```

Comment:
Since this is an intermediate-level tutorial, I suppose we also assume that the users already know some things like have a basic knowledge (or more than basic?) of NLP concepts. Do we need them to know anything else? basic knowledge of haystack? Anything else?

Reply:
I think having an installed instance of Haystack is a prerequisite as well

Reply:
@agnieszka-m I took it from Branden's [tutorial restructuring PR](https://github.com/deepset-ai/haystack-tutorials/pull/44/files#diff-dbe1a3c4d5cf3e90c8061c064829f918faf140fe28e11f6391d113e74c15aae7R14), assuming it to be a prerequisite for running the notebook and not as the prerequisite knowledge required for following the tutorial.

If you think users will be confused then I can add "Basic knowledge of Haystack (see Tutorial 1, 2)". 
Please let me know what you think

Reply:
I added the same comments to Branden's tutorials :) 
Yes, I think I'd add something like: This tutorial assumes basic knowledge of Haystack <a>Retrievers</a> and <a>Pipelines</a>. If you want to learn about them, have a look at "Title of Tutorial 1" and "Title of tutorial 2". " (of course the tutorial titles are links.)

Reply:
Added :)